scw/client.go

```diff
@@ -180,7 +188,12 @@ func (c *Client) do(req *ScalewayRequest, res interface{}) (sdkErr SdkError) {
 		if err != nil {
 			logger.Warningf("cannot dump outgoing request: %s", err)
 		} else {
-			logger.Debugf("dumping http request:\n" + string(dump))
+			var logString string
+			logString += "\n---------------- Scaleway SDK REQUEST %d : ---------------\n"
```

Comment:
Remove noise

Reply:
it is not really  readable without example:
```
2019/06/11 17:23:08 [DEBUG] creating POST request on https://api.scaleway.com/instance/v1/zones/fr-par-1/ips
2019/06/11 17:23:08 [DEBUG] creating POST request on https://api.scaleway.com/instance/v1/zones/fr-par-1/ips
2019/06/11 17:23:08 [DEBUG] dumping http request:
POST /instance/v1/zones/fr-par-1/ips HTTP/1.1
Host: api.scaleway.com
User-Agent: scaleway-sdk-go/0.0.0 (go1.12.5; darwin; amd64) terraform-provider/v1.10.0 terraform/0.12.1
Content-Length: 55
Content-Type: application/json
X-Auth-Token: 8dc1be15-xxxx-xxxx-xxxx-xxxxxxxxxxxx
Accept-Encoding: gzip

{"organization":"b3ba839a-dcf2-4b0a-ac81-fc32370052a0"}
2019/06/11 17:23:08 [DEBUG] dumping http request:
POST /instance/v1/zones/fr-par-1/ips HTTP/1.1
Host: api.scaleway.com
User-Agent: scaleway-sdk-go/0.0.0 (go1.12.5; darwin; amd64) terraform-provider/v1.10.0 terraform/0.12.1
Content-Length: 55
Content-Type: application/json
X-Auth-Token: 8dc1be15-xxxx-xxxx-xxxx-xxxxxxxxxxxx
Accept-Encoding: gzip

{"organization":"b3ba839a-dcf2-4b0a-ac81-fc32370052a0"}
[DEBUG] POST https://api.scaleway.com/instance/v1/zones/fr-par-1/ips
[DEBUG] POST https://api.scaleway.com/instance/v1/zones/fr-par-1/ips
2019/06/11 17:23:09 [DEBUG] dumping http response:
HTTP/2.0 201 Created
Content-Length: 171
Cache-Control: no-cache
Content-Security-Policy: default-src 'none'; frame-ancestors 'none'
Content-Type: application/json
Date: Tue, 11 Jun 2019 15:23:09 GMT
Location: https://cp-par1.scaleway.com/ips/14eec701-02dd-4294-9b70-ba6a5b382be5
Server: scaleway_api
Strict-Transport-Security: max-age=63072000
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{"ip": {"organization": "b3ba839a-dcf2-4b0a-ac81-fc32370052a0", "reverse": null, "id": "14eec701-02dd-4294-9b70-ba6a5b382be5", "server": null, "address": "51.15.245.155"}}
2019/06/11 17:23:09 [DEBUG] creating GET request on https://api.scaleway.com/instance/v1/zones/fr-par-1/ips/14eec701-02dd-4294-9b70-ba6a5b382be5
2019/06/11 17:23:09 [DEBUG] dumping http request:
GET /instance/v1/zones/fr-par-1/ips/14eec701-02dd-4294-9b70-ba6a5b382be5 HTTP/1.1
Host: api.scaleway.com
User-Agent: scaleway-sdk-go/0.0.0 (go1.12.5; darwin; amd64) terraform-provider/v1.10.0 terraform/0.12.1
X-Auth-Token: 8dc1be15-xxxx-xxxx-xxxx-xxxxxxxxxxxx
Accept-Encoding: gzip

[DEBUG] GET https://api.scaleway.com/instance/v1/zones/fr-par-1/ips/14eec701-02dd-4294-9b70-ba6a5b382be5
2019/06/11 17:23:09 [DEBUG] dumping http response:
HTTP/2.0 200 OK
Content-Length: 171
Cache-Control: no-cache
Content-Security-Policy: default-src 'none'; frame-ancestors 'none'
Content-Type: application/json
Date: Tue, 11 Jun 2019 15:23:09 GMT
Server: scaleway_api
Strict-Transport-Security: max-age=63072000
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{"ip": {"organization": "b3ba839a-dcf2-4b0a-ac81-fc32370052a0", "reverse": null, "id": "14eec701-02dd-4294-9b70-ba6a5b382be5", "server": null, "address": "51.15.245.155"}}
2019/06/11 17:23:09 [WARN] Provider "scaleway" produced an unexpected new value for scaleway_instance_ip.base, but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .project_id: was null, but now cty.StringVal("b3ba839a-dcf2-4b0a-ac81-fc32370052a0")
      - .reverse: was null, but now cty.StringVal("")
      - .zone: was null, but now cty.StringVal("fr-par-1")
2019/06/11 17:23:17 [DEBUG] dumping http response:
HTTP/2.0 201 Created
Content-Length: 171
Cache-Control: no-cache
Content-Security-Policy: default-src 'none'; frame-ancestors 'none'
Content-Type: application/json
Date: Tue, 11 Jun 2019 15:23:17 GMT
Location: https://cp-par1.scaleway.com/ips/f7bd849c-b250-4647-b0b7-775eeeacc18f
Server: scaleway_api
Strict-Transport-Security: max-age=63072000
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{"ip": {"organization": "b3ba839a-dcf2-4b0a-ac81-fc32370052a0", "reverse": null, "id": "f7bd849c-b250-4647-b0b7-775eeeacc18f", "server": null, "address": "51.15.223.163"}}
2019/06/11 17:23:17 [DEBUG] creating GET request on https://api.scaleway.com/instance/v1/zones/fr-par-1/ips/f7bd849c-b250-4647-b0b7-775eeeacc18f
2019/06/11 17:23:17 [DEBUG] dumping http request:
GET /instance/v1/zones/fr-par-1/ips/f7bd849c-b250-4647-b0b7-775eeeacc18f HTTP/1.1
Host: api.scaleway.com
User-Agent: scaleway-sdk-go/0.0.0 (go1.12.5; darwin; amd64) terraform-provider/v1.10.0 terraform/0.12.1
X-Auth-Token: 8dc1be15-xxxx-xxxx-xxxx-xxxxxxxxxxxx
Accept-Encoding: gzip

[DEBUG] GET https://api.scaleway.com/instance/v1/zones/fr-par-1/ips/f7bd849c-b250-4647-b0b7-775eeeacc18f
2019/06/11 17:23:17 [DEBUG] dumping http response:
HTTP/2.0 200 OK
Content-Length: 171
Cache-Control: no-cache
Content-Security-Policy: default-src 'none'; frame-ancestors 'none'
Content-Type: application/json
Date: Tue, 11 Jun 2019 15:23:17 GMT
Server: scaleway_api
Strict-Transport-Security: max-age=63072000
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{"ip": {"organization": "b3ba839a-dcf2-4b0a-ac81-fc32370052a0", "reverse": null, "id": "f7bd849c-b250-4647-b0b7-775eeeacc18f", "server": null, "address": "51.15.223.163"}}
2019/06/11 17:23:17 [WARN] Provider "scaleway" produced an unexpected new value for scaleway_instance_ip.scaleway, but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .zone: was null, but now cty.StringVal("fr-par-1")
      - .project_id: was null, but now cty.StringVal("b3ba839a-dcf2-4b0a-ac81-fc32370052a0")
      - .reverse: was cty.StringVal("www.scaleway.com"), but now cty.StringVal("")
2019/06/11 17:23:17 [DEBUG] provider has no plugin.Client
2019/06/11 17:23:17 [DEBUG] New state was assigned lineage "ff636baa-8367-6357-49a0-2f913287e9ab"
2019/06/11 17:23:17 [DEBUG] creating GET request on https://api.scaleway.com/instance/v1/zones/fr-par-1/ips/14eec701-02dd-4294-9b70-ba6a5b382be5
2019/06/11 17:23:17 [DEBUG] dumping http request:
GET /instance/v1/zones/fr-par-1/ips/14eec701-02dd-4294-9b70-ba6a5b382be5 HTTP/1.1
Host: api.scaleway.com
User-Agent: scaleway-sdk-go/0.0.0 (go1.12.5; darwin; amd64) terraform-provider/v1.10.0 terraform/0.12.1
X-Auth-Token: 8dc1be15-xxxx-xxxx-xxxx-xxxxxxxxxxxx
Accept-Encoding: gzip

[DEBUG] GET https://api.scaleway.com/instance/v1/zones/fr-par-1/ips/14eec701-02dd-4294-9b70-ba6a5b382be5
2019/06/11 17:23:18 [DEBUG] dumping http response:
HTTP/2.0 200 OK
Content-Length: 171
Cache-Control: no-cache
Content-Security-Policy: default-src 'none'; frame-ancestors 'none'
Content-Type: application/json
Date: Tue, 11 Jun 2019 15:23:17 GMT
Server: scaleway_api
Strict-Transport-Security: max-age=63072000
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{"ip": {"organization": "b3ba839a-dcf2-4b0a-ac81-fc32370052a0", "reverse": null, "id": "14eec701-02dd-4294-9b70-ba6a5b382be5", "server": null, "address": "51.15.245.155"}}
2019/06/11 17:23:18 [DEBUG] creating GET request on https://api.scaleway.com/instance/v1/zones/fr-par-1/ips/f7bd849c-b250-4647-b0b7-775eeeacc18f
2019/06/11 17:23:18 [DEBUG] dumping http request:
GET /instance/v1/zones/fr-par-1/ips/f7bd849c-b250-4647-b0b7-775eeeacc18f HTTP/1.1
Host: api.scaleway.com
User-Agent: scaleway-sdk-go/0.0.0 (go1.12.5; darwin; amd64) terraform-provider/v1.10.0 terraform/0.12.1
X-Auth-Token: 8dc1be15-xxxx-xxxx-xxxx-xxxxxxxxxxxx
Accept-Encoding: gzip

[DEBUG] GET https://api.scaleway.com/instance/v1/zones/fr-par-1/ips/f7bd849c-b250-4647-b0b7-775eeeacc18f
2019/06/11 17:23:18 [DEBUG] dumping http response:
HTTP/2.0 200 OK
Content-Length: 171
Cache-Control: no-cache
Content-Security-Policy: default-src 'none'; frame-ancestors 'none'
Content-Type: application/json
Date: Tue, 11 Jun 2019 15:23:18 GMT
Server: scaleway_api
Strict-Transport-Security: max-age=63072000
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

{"ip": {"organization": "b3ba839a-dcf2-4b0a-ac81-fc32370052a0", "reverse": null, "id": "f7bd849c-b250-4647-b0b7-775eeeacc18f", "server": null, "address": "51.15.223.163"}}
2019/06/11 17:23:18 [WARN] Test: Executing destroy step
```

Reply:
especially when request are made concurrently